docs/acceptance-criteria/uacs-rfc-19-random-beacon-v2.adoc

```diff
@@ -0,0 +1,467 @@
+  misbehaving members (members who were inactive or were disqualified) and
+  should be submitted by eligible group member on-chain.
+
+| Group creation - DKG submission
+| When determining validity of the tx submitting the DKG result, the following
+  aspects should be taken into account: +
+  - is the submitter eligible at the moment to send the tx? +
+  - are there enough supporting signatures on the result? +
+  - do the signatures come from the stakers with at least a minimum stake? +
+  Only if all the above conditions have been met, the tx is considered valid.
+
+| Group creation - DKG submission
+| The transaction submitting DKG result should have a predictable cost.
+
+| Group creation - DKG submission
+| At a given moment, only selected members from the group should be eligible to
+  successfully submit the DKG result to the chain: +
+  - at the beginning (right after the challenge period passes) only one member
+  with index `hash(new_group_pubkey) % group_size` should be eligible +
+  - after fixed amount of blocks another member (with index
+  `(hash(new_group_pubkey) % group_size ) + 1` should become eligible (previous
+  operator remains eligible) +
+  - next members should become eligible after each fixed amount of blocks.
+
+| Group creation - DKG submission
+| If random beacon deems the transaction submitting the DKG valid, then: +
+  - the DKG result submission eligibility period should finish (all other
+  results should be rejected from now on) +
+  - a challenge period should start.
+
+| Group creation - DKG challenge
+| Anyone can send a challenge notification that submitted DKG result contains
+  group members not selected by the pool.
+
+| Group creation - DKG challenge
+| Upon processing the challenge notification, the random beacon contract should
+  determine validity of the challenge.
+
+| Group creation - DKG challenge
+| When determining validity of the challenge, the following aspects should be
+  taken into account: +
+  - does it reference existing DKG result (if not, then challenge invalid) +
+  - within or outside of the challenge period for the specified DKG result (if
+  outside, then invalid) +
+  - have all members who signed a DKG result been selected by the sortition pool
+  to be in the group (if yes, then challenge invalid).
+
+| Group creation - DKG challenge
+| If the challenge notification was received within the challenge period and was
+  deemed justified, then: +
+  - the malicious DKG result should be immediately discarded (in the same
+  transaction in which notification happened) +
+  - all sortition pool members who signed the result should be slashed (fixed
+  amount) (in the same tx) +
+  - notifier should receive 5% from the total slashed amount (in the same tx) +
+  - the members of the signing group should be given another chance to publish
+  the DKG result
+  - DKG timeout timer and the result subission eligibility order shoul be reset.
```

Comment:
Some typos: 
subission -> submission
shoul -> should

Reply:
Fixed, see 7cc86c2e707b6b13450709e6f596f933b4c2439c.